File: Python/EjercicioExtras/01_calculadora.py
opciones = int(input("""
1. SUMA
2. RESTA
3. MULTIPLICACIOM
4. DIVISION
Elige entre 1 - 4: """))

ingresePrimerNumero = float(input("Ingrese primer número: "))
ingreseSegundoNumero = float(input("Ingrese segundo número: "))
#resultado = 0


def calculadora(case, num1, num2 ):
    if case == 1:
        return num1 + num2

    elif case == 2:
        return num1 - num2

    elif case == 3:
        return num1 * num2

    elif case == 4:
        if num2 == 0:
            print("No puedes dividir entre CERO.")
        else:
            return num1 / num2
    else:
        return "ERROR - Elegige entre 1 - 4"

resultado = calculadora(opciones, ingresePrimerNumero, ingreseSegundoNumero)
print(f"El resultado es {resultado}")

File: JavaScript/EjercicioExtras/01_calculadora.js
let calculadora =  parseInt(prompt("1. suma\n2. resta\n3. multiplicacion\n4. division"))


let ingresePrimerNumero = parseFloat(prompt("Ingrese primer número: "))
let ingreseSegundoNumero = parseFloat(prompt("Ingrese segundo número: "))
let resultado = 0

switch (calculadora) {
    case 1:
        resultado = ingresePrimerNumero + ingreseSegundoNumero
        document.write(`El resultado es ${resultado}`)
        break;

    case 2:
        resultado = ingresePrimerNumero - ingreseSegundoNumero
        document.write(`El resultado es ${resultado}`)
        break;

    case 3:
        resultado = ingresePrimerNumero * ingreseSegundoNumero
        document.write(`El resultado es ${resultado}`)
        break;

    case 4:
        if (ingreseSegundoNumero == 0) {
            document.write("ERRO - no se puede dividir entre CERO.")
        }
        else{
            resultado = ingresePrimerNumero / ingreseSegundoNumero
            document.write(`El resultado es ${resultado}`)
        }

        break;

    default:
        document.write(`Debes elegir entre las 1 - 4`)
        break;
}